src/test/java/CarComfortServiceTest.java

```diff
@@ -0,0 +1,243 @@
+        UUID id = UUID.randomUUID();
+        String exceptionMessage = "Car comfort with such an id not found";
+
+        when(carComfortRepositoryMock.getById(id)).thenReturn(null);
+        CarRentalException exception = assertThrows(
+                CarRentalException.class, () -> carComfortService.getById(id));
+
+        assertNotNull(exception);
+        assertEquals(exceptionMessage, exception.getMessage());
+
+        verify(carComfortRepositoryMock).getById(id);
+        verifyNoMoreInteractions(carComfortRepositoryMock);
+    }
+
+
+    @Test
+    public void whenUpdateCorrectCarComfort_thenVerifyCarComfort() {
+        CarComfort carComfort = buildCarComfort(UUID.randomUUID());
+
+        when(carComfortRepositoryMock.update(carComfort)).thenReturn(carComfort);
+
+        CarComfort response = carComfortService.update(carComfort);
+        assertNotNull(response);
+
+        assertEquals(carComfort.getId(), response.getId());
+        assertEquals(carComfort.getName(), response.getName());
+        assertEquals(carComfort.getDescription(), response.getDescription());
+
+        verify(carComfortRepositoryMock).update(carComfort);
+        verifyNoMoreInteractions(carComfortRepositoryMock);
+    }
+
+
+    @Test
+    public void whenUpdateNullCarComfort_thenVerifyException() {
+        String exceptionMessage = "Car comfort must be NOT null";
+
+        CarRentalException exception = assertThrows(
+                CarRentalException.class, () -> carComfortService.update(null));
+        assertNotNull(exception);
+        assertEquals(exceptionMessage, exception.getMessage());
+
+        verifyNoInteractions(carComfortRepositoryMock);
+    }
+
+
+    @Test
+    public void whenUpdateInvalidCarComfort_thenVerifyException() {
+        CarComfort carComfort = new CarComfort();
+
+        String exceptionMessage = "Car comfort has invalid data";
+        CarRentalException exception = assertThrows(
+                CarRentalException.class, () -> carComfortService.update(carComfort));
+        assertNotNull(exception);
+        assertEquals(exceptionMessage, exception.getMessage());
+
+        verifyNoInteractions(carComfortRepositoryMock);
+    }
+
+
+    @Test //???
+    public void whenDeleteWithInvalidId_thenVerifyException() {
+        String exceptionMessage = "Can't remove the сar comfort, its id must be NOT null";
+
+        CarRentalException exception = assertThrows(
+                CarRentalException.class, () -> carComfortService.delete(null));
+        assertNotNull(exception);
+        assertEquals(exceptionMessage, exception.getMessage());
+
+        verifyNoInteractions(carComfortRepositoryMock);
+        /*doThrow(new CarRentalException("Can't remove the сar comfort, its id must be NOT null"))
+                .when(carComfortRepositoryMock)
+                .delete(null);
+        assertThrows(CarRentalException.class, ()->carComfortRepositoryMock.delete(null));*/
+    }
```

Comment:
You do not need doThrow here. Your test without comments looks good

Reply:
Done